Index: b/gyp_uv.py
===================================================================
--- a/gyp_uv.py
+++ b/gyp_uv.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2
 
 import os
 import platform
Index: b/tools/make_dist_html.py
===================================================================
--- a/tools/make_dist_html.py
+++ b/tools/make_dist_html.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python2
 
 from __future__ import print_function
 
